src/Vogen/Templates/Double/Double_SystemTextJsonConverter.cs

```diff
@@ -3,13 +3,11 @@ class VOTYPESystemTextJsonConverter : global::System.Text.Json.Serialization.Jso
         {
             public override VOTYPE Read(ref global::System.Text.Json.Utf8JsonReader reader, global::System.Type typeToConvert, global::System.Text.Json.JsonSerializerOptions options)
             {
-__NORMAL__                return VOTYPE.From(reader.GetDouble());
-__STRING__                return VOTYPE.From(global::System.Double.Parse(reader.GetString(), global::System.Globalization.NumberStyles.Any, global::System.Globalization.CultureInfo.InvariantCulture));
+                return VOTYPE.From(reader.GetDouble());
             }
 
             public override void Write(System.Text.Json.Utf8JsonWriter writer, VOTYPE value, global::System.Text.Json.JsonSerializerOptions options)
             {
-__NORMAL__                writer.WriteNumberValue(value.Value);
-__STRING__                writer.WriteStringValue(value.Value.ToString(global::System.Globalization.CultureInfo.InvariantCulture));
+                writer.WriteNumberValue(value.Value);
```

Comment:
Same comment as above; we need to keep the placeholders - one of them is removed when the source is generated (they're used to give users the ability to treat numbers as strings.